### PR TITLE
Fixes #1070

### LINKED ISSUE #1070: Falsy schema defaults not output as expected
Falsy schema defaults i.e. `false`, `""` and `0` are ignored and attributes are set to `null` in the outputted JSON until the corresponding Backbone Form is submitted.

### PR DESCRIPTION
**Pull request by tomgreenfield**

### CODE REVIEW

File: lib/helpers.js [hunk 1]
```diff
@@ -24,18 +24,24 @@ function schemaToObject (schema, id, version, location) {
...
-      switch (props[key].type) {
+      var prop = props[key];
+
+      switch (prop.type) {
         case "boolean":
+          child[key] = prop.default || false;
+          break;
         case "integer":
         case "number":
+          child[key] = +prop.default || 0;
```
Comment (dennis-learningpool) [inline lib/helpers.js]: should this use parseInt?

Comment (dennis-learningpool) [inline lib/helpers.js]: ... nah

**Comment by dennis-learningpool** [comment]
+1

**Comment by brian-learningpool** [comment]
+1

**Comment by taylortom** [comment]
+1

**Pull request merged**